### PR TITLE
features: dpdk build: Check for more states in post deploy script

### PR DESCRIPTION
**Pull request by sabinaaledort**
We were missing few states. Exit with error if didn't match to Complete state.

### CODE REVIEW

File: feature-configs/deploy/s2i/post_deploy.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ else
     build_status=$(oc get build -n dpdk $last_build -o json | jq '.status.phase' | tr -d '"')
     if [ $build_status == "Complete" ]; then
         exit 0
-    elif [ $build_status == "Running" ]; then
+    elif [ $build_status == "Running" ] || [ $build_status == "Pending" ]; then
```
Comment (zeeke) [inline feature-configs/deploy/s2i/post_deploy.sh:11]: nit: also missing `New` and `Cancelled`. What about putting `exit 1` at the end of the script to catch all unwanted build status?

https://docs.okd.io/4.9/rest_api/workloads_apis/build-build-openshift-io-v1.html#specification

Comment (sabinaaledort) [reply feature-configs/deploy/s2i/post_deploy.sh:11]: nice! i didn't know there more statuses. thanks! i will add it

**Comment by SchSeba** [comment]
OCP 4.12 change the PSP to enforce :(

```
Error from server (Forbidden): error when creating "STDIN": pods "podman" is forbidden: violates PodSecurity "baseline:latest": privileged (container "priv" must not set securityContext.privileged=true)
/go/src/github.com/openshift-kni/cnf-features-deploy
make: *** [Makefile:103: setup-build-index-image] Error 1
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-07-27T11:19:04Z"}
error: failed to execute wrapped command: exit status 2 
```

I will need to work on this

**Pull request merged**